test/t8_IO/t8_gtest_vtk_reader.cxx

```diff
@@ -36,7 +36,24 @@ const vtk_file_type_t gtest_vtk_filetypes[VTK_NUM_TYPES]
   = { VTK_FILE_ERROR, VTK_UNSTRUCTURED_FILE, VTK_POLYDATA_FILE, VTK_PARALLEL_UNSTRUCTURED_FILE,
       VTK_PARALLEL_POLYDATA_FILE };
 
-class vtk_reader: public t8_test_with_attributes<testing::TestWithParam<std::tuple<int, int, int>>> {
+class vtk_reader: public testing::TestWithParam<std::tuple<int, int, int>> {
+ public:
+  static void
+  SetUpTestSuite ()
+  {
+    t8_debugf ("[D] Register package \n");
+    t8_package_id = sc_package_register (NULL, SC_LP_DEFAULT, "GoogleTest",
+                                         "t8code testsuite package. Used for testing of external user attributes.");
```

Comment:
Since this is explicetly not the t8code package but an additional package that is register, we should not name the variable "t8_package_id"

Suggestion: "t8_testsuite_package_id" or "t8_gtest_package_id"